app/models/state_file_base_intake.rb

```diff
@@ -78,16 +78,21 @@ def self.selected_intakes_for_deadline_reminder_notifications
       .where(efile_submissions: { id: nil })
       .where.not(df_data_imported_at: nil)
       .has_verified_contact_info
-      .select(&:should_not_be_sent_reminder)
-  end
-
-  def should_not_be_sent_reminder
-    if message_tracker.present? && message_tracker["messages.state_file.finish_return"]
-      finish_return_msg_sent_time = Time.parse(message_tracker["messages.state_file.finish_return"])
-      finish_return_msg_sent_time < 24.hours.ago
-    else
-      !disqualifying_df_data_reason.present?
-    end
```

Comment:
having this as an if/else meant that if you had received a message more than 24 hours ago but also had disqualifying data, you'd still receive the message, which is wrong